app/views/docs/migration-firebase.phtml

```diff
@@ -0,0 +1,3 @@
+<p>
+    [TODO: Bradley try to fill this in with necessary information + limitations]
```

Comment:
@PineappleIOnic 

This document should contain everything a user needs to know about migrating from Firebase to Appwrite, don't worry about looking pretty, get the information down first, we'll edit this and use flow as template for other providers